packages/server/tsconfig.json

```diff
@@ -4,5 +4,5 @@
     "module": "NodeNext",
     "moduleResolution": "NodeNext"
   },
-  "include": ["src/**/*.ts"]
+  "include": ["src/**/*.ts", "src/**/*.json"]
```

Comment:
Needed to add `.json` so that `data-version-manifest.json` is included in the `dist`